knx/knxnet/proto.go

```diff
@@ -139,11 +141,11 @@ func Unpack(data []byte, srv *Service) (uint, error) {
 		return n, err
 	}
 
-	if headerLen != 6 {
+	if headerLen != 0x6 {
```

Comment:
I don't understand this change. Why?

Reply:
The specification uses hex length, would make it easier to read. But reverted to integers.